examples/v2/declarative_workflows/worker.py

```diff
@@ -0,0 +1,63 @@
+from collections import Counter
+from typing import Literal
+
+from examples.v2.declarative_workflows.client import hatchet
+from examples.v2.declarative_workflows.workflows import (
+    Greeting,
+    Language,
+    greet_workflow,
+    language_counter_workflow,
+)
+from hatchet_sdk import Context
+
+
+def complete_greeting(greeting: Greeting) -> str:
+    match greeting:
+        case "Hello":
+            return "world!"
+        case "Ciao":
+            return "mondo!"
+        case "Hej":
+            return "världen!"
+
+
+@greet_workflow.declare()
+async def greet(ctx: Context) -> dict[Literal["message"], str]:
+    workflow_input = greet_workflow.workflow_input(ctx)
+    greeting = workflow_input.greeting
```

Comment:
these type check correctly, so no need to `cast` or parse from a dict to an `ExampleInput`